### ISSUE TITLE
modern technology stack

### ISSUE THREAD

**Issue by ktabor** [18F/bpa-opm-eqip#52]
# Question/Comment on e-QIP Prototype RFQ
Fifth Question 

## Name and affiliation
Katherine Tabor
Senior Program Analyst 
I assist with proposal development 

## Section of RFQ documents
PWS.  Section 5.0 (Tasks).  Second bullet - Contractor shall build a front-end software application prototype of the e-QIP system using a modern technology stack, digitizing the OPM Standard Form 86 (SF-86) or its successor and associated questions.

## Question/Comment
Does the Government have any preferences for the “modern technology stack” used for this solution outside the open source parameters identified in the solicitation?

**Comment by jehlers** [comment]
No. However, it will be hosted on the cloud.gov PaaS and therefore must be compatible. You may find the cloud.gov documentation here: https://docs.cloud.gov/
